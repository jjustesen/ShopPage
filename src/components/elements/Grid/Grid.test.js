import React from "react";

import { render, cleanup } from "@sponte/lib-utils/dist/test-utils";

import { RstRow, RstCol } from ".";

afterEach(cleanup);

describe("RstRow and RstCol", () => {
  it("deve renderizar e ser compativel com a versao anterior", () => {
    const { asFragment } = render(
      <RstRow>
        <RstCol>Sponte</RstCol>
      </RstRow>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
