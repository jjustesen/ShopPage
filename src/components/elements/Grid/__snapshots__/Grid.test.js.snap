// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RstRow and RstCol deve renderizar e ser compativel com a versao anterior 1`] = `
<DocumentFragment>
  <div
    class="sc-gzVnrw dPmAXc"
  >
    <div />
  </div>
  <div
    class="sc-dnqmqq sc-gZMcBi sc-gqjmRU ecAJdE"
    display="flex"
  >
    <div
      class="sc-dnqmqq sc-VigVT kBNxyI"
      width="1"
    >
      Sponte
    </div>
  </div>
</DocumentFragment>
`;
